docker/Dockerfile

```diff
@@ -19,7 +19,7 @@
 # For those usages not covered by the GNU Affero General Public License
 # please contact with: [daniel.moranjimenez@telefonica.com]
 
-ARG NODE_VERSION=10.17.0-slim
+ARG NODE_VERSION=10.19.0-slim
```

Comment:
CHANGES_NEXT_RELEASE line

> Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life 

Should be updated (10.17.0 -> 10.19.0)

Reply:
Fixed a0b5a27